components/User/ProductMenuItem.tsx

```diff
@@ -21,23 +20,36 @@ const ProductMenuItem = ({
     <>
       <Flex
         width="100%"
+        flexDirection="column"
         alignItems="center"
-        gap="17px"
+        gap="13px"
         onClick={() => router.push(routingUrl)}
       >
-        <Image src={iconUrl} alt={`${title} 아이콘`} />
+        <Image
+          width="33.92px"
+          height="38px"
+          src={iconUrl}
+          alt={`${title} 아이콘`}
+        />
         <Text
           flexGrow="1"
+          color="brand.dark"
           fontFamily="Roboto"
           fontStyle="normal"
           fontWeight="400"
-          fontSize="16"
+          fontSize="14"
+          lineHeight="128.19%"
         >
           {title}
         </Text>
-        <ChevronRightIcon justifySelf="flex-end" />
       </Flex>
-      {!isLastItem ? <Divider /> : undefined}
+      {!isLastItem ? (
+        <Divider
+          orientation="vertical"
+          border="3px solid #EFEFEF"
+          height="20px"
+        />
+      ) : undefined}
```

Comment:
`!isLastItem`이 false일때 반환되는 값이 없어서 아래와 같이 작성하는건 어떻게 생각하시나요??
```suggestion
{isLastItem && (
  <Divider
    orientation="vertical"
    border="3px solid #EFEFEF"
    height="20px"
  />
)}
```

Reply:
아주 좋습니다! 👍